File: src/main/java/cn/labzen/meta/meta/MetaMeta.java
package cn.labzen.meta.meta;

import cn.labzen.meta.component.DeclaredComponent;

public class MetaMeta implements DeclaredComponent {

  @Override
  public String mark() {
    return "Labzen-Meta";
  }

  @Override
  public String packageBased() {
    return "cn.labzen.meta";
  }

  @Override
  public String description() {
    return "收集、管理、报告Labzen组件的元信息";
  }
}

File: src/main/kotlin/cn/labzen/meta/meta/MetaMeta.kt
package cn.labzen.meta.meta

import cn.labzen.meta.component.LabzenComponent

class MetaMeta : LabzenComponent {

  override fun mark(): String =
    "Labzen-Meta"

  override fun packageBased(): String =
    "cn.labzen.meta"

  override fun description(): String =
    "收集、管理、报告Labzen组件的元信息"
}
